modules/create_timepoint/jsx/createTimepointIndex.js

```diff
@@ -122,13 +123,12 @@ class CreateTimepoint extends React.Component {
     ).then((response) => response.json())
       .then(
         (data) => {
-          console.log(data);
           // Populate the form errors.
           if (data.errors) {
-            // data for the form errors.
-            this.setState({errors: data.errors});
-            this.setState({isLoaded: true});
-            return;
+            const state = Object.assign({}, this.state);
+            state.errors = data.errors;
+            state.form.disabled = true;
```

Comment:
could you explain what you're doing here with disabling the button? I can't trigger this case on the front-end

Reply:
Thanks @zaliqarosli, that's code that isn't needed anymore since your help with the unfilled form errors.